frontend/src/app/main/refs.cljs

```diff
@@ -361,6 +360,10 @@
 (def workspace-focus-selected
   (l/derived :workspace-focus-selected st/state))
 
+;; Remove this when deprecating components-v2
```

Comment:
the components-v2 will be deprecated?

Reply:
At some moment in the future we should delete all old components code, and remove the check of components-v2 (this adds a lot of complexity that may cause errors). I add this comment here so we can do a search for "components-v2" and locate this function, that can be removed.